Data/Services/CustomerService.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Data.Entities;
+using Data.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Data.Repositories
+{
+    public class CustomerService : ICustomerService
+    {
+        private readonly List<Customer> _customers = new();
+
+        public void Create(Customer createdCustomer)
+        {
+            bool emailAlreadyExists = _customers.Any(customer => customer.Email == createdCustomer.Email);
+            if (emailAlreadyExists)
+                throw new ArgumentException("Email already exists");
+
+            bool cpfAlreadyExists = _customers.Any(customer => customer.Cpf == createdCustomer.Cpf);
+            if (cpfAlreadyExists)
+                throw new ArgumentException("Cpf already exists");
+            
+
+            createdCustomer.Id = _customers.LastOrDefault()?.Id + 1 ?? 1; 
+
+            _customers.Add(createdCustomer);
+        }
+
+        public bool Delete(long id)
```

Comment:
```suggestion
        public void Delete(long id)
```
Poderia ser um void, pois o método só executa a remoção de um customer quando existe um com Id valido.